project_event/models/project_task.py

```diff
@@ -265,7 +265,12 @@ def create_new_activity(self, vals):
     @api.multi
     def write(self, vals):
         if self.activity_task_type == 'activity':
-            self.write_task(vals)
+            if 'task_state' in vals:
+                state = vals.pop('task_state')
```

Comment:
@eilst : delete line 269 and change the line 271 as   vals['task_state'] = vals.pop('task_state').

Reply:
@neophilic_insect07  Done, I put this on the write_main_task method